.travis.yml

```diff
@@ -5,10 +5,10 @@ python:
   - "3.8"
 install:
   - pip install --upgrade pip
-  - pip install numpy jaxlib tensorflow tensorflow-datasets --upgrade
+  - pip install pygame==2.0.0.dev6 tfp-nightly tfds-nightly numpy jaxlib jax tf-nightly matplotlib more-itertools --upgrade
   - pip install git+https://github.com/google/jax.git
   - pip install -e .
+  - cp tf_helpers/lax.py tests/
```

Comment:
Did you intend to copy the test file here? If not, I'm not sure why the cp, and how does the new test get tested? 

Could any copying be avoided by e.g. putting the file on the tests folder in the first place?

Reply:
Hi Roman, as you mentioned, I will put the `lax_test.py` file under tests folder. In terms of `lax` and `tf_jax_stax` (`ostax`) that I am going to commit later on, what do you think is a good place to put them? Actually only the test files will directly access them so it definitely will work if directly putting them under the tests folder. However, this does not look like a good structure. If we keep it under `tf_helpers`, maybe there is a need to integrate them as part of Neural Tangents in order to import them? (I am not sure doing something like `from ...tf_helpers import lax` will work). I need your insight here. Thanks!

Reply:
I believe having a top-level folder (same level as neural_tangents and tests) `tf_helpers`, and then in tests calling something like `from tf_helpers import lax` would work, or not? You may need an empty `__init__.py` file inside the `tf_helpers` folder, not an expert in module organization tbh. Re structure, I would be OK with either a separate top-level `tf_tests` / have files inside `tests` with `tf_` suffixes / or just have them in `tests`, and rename/move around if we run into naming conflicts / confusion. Since I don't see the whole structure very clearly yet, it may be best for you to decide! But I just find copying files a bit awkward and would instead try to call them from their original location.